js/index.js

```diff
@@ -1 +1,127 @@
 // Your code goes here
+
+
+const funBusImg = document.querySelector(".intro img")
+const welcome = document.querySelector(".intro h2")
+const body = document.querySelector("body")
+const firstParagraph = document.querySelector(".intro p")
+const header = document.querySelector(".main-navigation")
+const adventureImg = document.querySelector(".img-content img")
+const destinationImg = document.querySelector(".content-destination.img")
+const funImg = document.querySelector(".inverse-content img")
+const footer = document.querySelector(".footer p")
+const destinationTxt = document.querySelector("textarea")
+const btn = document.querySelectorAll(".btn")
+const destinationH4 = document.querySelectorAll(".content-pick .destination")
+const h2 = document.querySelectorAll("h2")
```

Comment:
I like that you are separating your concerns here, but in this case for readability I would recommend plugging these in next to their corresponding event listener